src/pages/PlaylistManagmentPage/IndividualPlayList/IndividualPlayList.jsx

```diff
@@ -28,9 +29,14 @@ function IndividualPlayList() {
   return (
     <div>
       <div className='playlist-individual-container'>
-        {PlayListVideos?.map((item) => (
+        {PlayListVideos.length === 0 
+          ? <div className='nocontent'>
+            <div className='page-title md-txt' >The Playlist Contains No Video </div><img src={HolderImg6} className="holders" /></div>
+        :  PlayListVideos?.map((item) => (
           <PlayListVideoCards key={item.id} props={item} setPlayListVideos={setPlayListVideos} />
```

Comment:
Try to check these conditions outside JSX for clean and easy to review code.

Reply:
I think this is on rerendering whenever updated and moving outside will create not required variable to be updated on rerenders as in I find this a better thing instead of creating new variable